.travis.yml

```diff
@@ -2,25 +2,36 @@ language: python
 
 sudo: false
 
-env:
-    - PYTHON="2.7"
-    - PYTHON="3.4"
-    - PYTHON="3.5"
+matrix:
+  fast_finish: true
+  include:
+  - python: 2.7
+    env: TEST_TARGET=default
+  - python: 3.5
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=default
+  - python: 3.6
+    env: TEST_TARGET=coding_standards
+
 
 before_install:
-    - wget http://bit.ly/miniconda -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda config --add channels ioos -f
-    - conda update --yes --all
-    - travis_retry conda create --yes -n test python=$PYTHON --file requirements.txt
-    - source activate test
-    - conda install --yes --file requirements-dev.txt;
-    - if [[ "$PYTHON" == "3.4" ]]; then
-        travis_retry conda install --yes nbconvert jupyter_client ipykernel;
-      fi
-    - travis_retry pip install
+  - wget http://bit.ly/miniconda -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --add channels conda-forge --force
```

Comment:
@knil-sama here is where we define the channels we will use, but instead of the old "in-house" `ioos` solution I had, or the third party `bokeh`, we will use the community channel `conda-forge`.